lib/api/core/auth/passwordManager.js

```diff
@@ -0,0 +1,21 @@
+var crypto = require('crypto');
+
+module.exports = function PassportManager () {
+
+  this.secret = 'jbsB8r69PFP39KdLtjVr25Z22';
```

Comment:
Ahh ok!
I couldn't understand what that meant, but it's now clearer. Can you add a TODO comment then?